registry/data-models/models.py

```diff
@@ -0,0 +1,152 @@
+from registry.data-models.transformation.models import *
+from registry.data-models.common.models import SemanticVersion, FeathrModel, Function
+from typing import Optional
+from pydantic import BaseModel
+
+
+"""
```

Comment:
We created this file before right in the previous PR? How come this is showing us as a new file rather than updates on the old one?

Reply:
I had some conflicts earlier. I just resolved all the conflicts, diff should be right now.